experimental/generation/generator/packages/library/templates/standard/en-us/dynamiclistEntity.en-us.lg.lg

```diff
@@ -0,0 +1,52 @@
+[standard.en-us.lg](standard.en-us.lg)
+
+# filename
+- ${entityLGFilename()}
+
+# template
+-```
+${formImport('form.en-us.lg')}
+${formImport('form-Choose.en-us.lg')}
+${propertyEntityValue()}
+${propertyEnumValue()}
+${chooseEntity()}
+```
+
+# chooseEntity
+-```
+# ${entity}_chooseEntity()
+> Choose between '${entity}' values
+- \${chooseEnumEntity()}```
+
+# propertyEntityValue
+-```
+# ${entity}_Value(entity) 
+> Display value for '${entity}'
+- IF: \${isArray(entity.value)}
+    - \${join(foreach(entity.value, result, `${result} `), 'and')}
+- ELSE:
+    - \${entity.value}```
+
+# propertyEnumValue
+- ${join([namePlusSwitch(), cases(), default()], '')}
+
+# namePlusSwitch
+-```
+# ${enumName(entity)}(value) 
+> Display value for '${entity}' enumeration value
+- SWITCH: \${value}```
+
+# case(option)
+-```
+  - CASE: \${'${option}'} 
+    - ${phrase(option)}```
+
+# cases
+- ${join(foreach(indicesAndValues(examples), enum, case(enum.index)), '')}
+
+# default
+-```
+  - DEFAULT: 
+    - no value
+```
```

Comment:
You don't need this any more.  This only handles the "static: examples, not the dynamic ones.